src/status_im/contexts/wallet/send/routes/view.cljs

```diff
@@ -1,39 +1,136 @@
-   (if (= status :default)
-     [quo/network-link
-      {:shape           :linear
-       :source          from-network
-       :destination     to-network
-       :container-style style/network-link}]
-     [rn/view {:style {:width 73}}])
-   [quo/network-bridge
-    {:amount          amount
-     :network         to-network
-     :status          status
-     :container-style {:right 12}}]])
+  [{:keys [amount from-network to-network status theme fetch-routes]}]
+  (if (= status :add)
+    [quo/network-bridge
+     {:status          :add
+      :container-style style/add-network
+      :on-press        #(rf/dispatch [:show-bottom-sheet
+                                      {:content (fn [] [networks-drawer
+                                                        {:theme        theme
+                                                         :fetch-routes fetch-routes}])}])}]
+    [rn/view {:style style/routes-inner-container}
+     [quo/network-bridge
+      {:amount  amount
+       :network from-network
+       :status  status}]
+     (if (= status :default)
+       [quo/network-link
+        {:shape           :linear
+         :source          from-network
+         :destination     to-network
+         :container-style style/network-link}]
+       [rn/view {:style {:width 73}}])
+     [quo/network-bridge
+      {:amount          amount
+       :network         to-network
+       :status          status
+       :container-style {:right 12}}]]))
 
-(defn view
-  [{:keys [amount routes loading-networks]}]
-  (let [loading-suggested-routes? (rf/sub [:wallet/wallet-send-loading-suggested-routes?])
-        candidates                (:candidates routes)]
-    (if (or (and (not-empty loading-networks) loading-suggested-routes?) (not-empty candidates))
+(defn- find-affordable-networks
+  [{:keys [balances-per-chain]} input-value selected-networks]
+  (->> balances-per-chain
+       (filter (fn [[_ {:keys [balance chain-id]}]]
+                 (and
+                  (>= (js/parseFloat balance) input-value)
+                  (some #(= % chain-id) selected-networks))))
+       (map first)))
+
+(defn- view-internal
+  [{:keys [amount routes token input-value theme fetch-routes]}]
+  (let [selected-networks         (rf/sub [:wallet/wallet-send-selected-networks])
+        loading-networks          (find-affordable-networks token input-value selected-networks)
+        loading-suggested-routes? (rf/sub [:wallet/wallet-send-loading-suggested-routes?])
+        best-routes               (:best routes)
+        data                      (if loading-suggested-routes? loading-networks best-routes)]
+    (if (or (and (not-empty loading-networks) loading-suggested-routes?) (not-empty best-routes))
       [rn/flat-list
-       {:data                    (if loading-suggested-routes? loading-networks candidates)
+       {:data                    (if (and (< (count data) 3) (pos? (count data)))
+                                   (concat data [{:status :add}])
```

Comment:
Do we care about the order here? consider using `conj` depending on the type of `data`

Reply:
I tried using `conj` but it adds it to the beginning. I want `{:status :add}` to be at the end.